ci-operator/step-registry/tcpdump-service/tcpdump-service-commands.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Injecting manifest to deploy tcpdump service MachineConfig with script and systemd units"
+
+echo
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+sysd_script=$(cat << EOF
+#!/bin/sh
+
+set -e
+
+# Build up a tcpdump filter for all quay.io IPs. These change seemingly daily
+# so by doing the lookup before we launch we increase our chances of dumping
+# the packets we're after.
+quay_lines=\$(dig quay.io | grep "^quay.io\." | awk '{print \$5}')
+tcpdump_filter=""
+for i in \$quay_lines
+do
+    if [ -z "\$tcpdump_filter" ]
+    then
+        tcpdump_filter+="(host \$i"
+    else
+        tcpdump_filter+=" or host \$i"
+    fi
+done
+# Add in registry.ci.openshift.org IPs as of 2021-12-22 and close the filter:
+tcpdump_filter+=" or host 3.210.253.73 or host 3.234.78.82)"
+
+echo "tcpdump filter: \$tcpdump_filter"
+echo
+echo "Running tcpdump:"
+echo
+
+/usr/sbin/tcpdump -i any -s 256 -w '/var/log/tcpdump/tcpdump.pcap' \$tcpdump_filter
```

Comment:
Add `-nn  -U` to the arguments.  No need for any name resolution, and avoid buffered output.